.travis.yml

```diff
@@ -48,8 +48,14 @@ matrix:
     install:
     - >
       docker exec --tty "$containerid" apt-get -y --no-install-recommends install
-      python3.5-dev python3-scipy libeigen3-dev python3-sphinx python3-sphinx-rtd-theme
+      python3.5-dev python3-scipy libeigen3-dev
       cmake make g++
+  # Documentation build:
+  - os: linux
+    language: docs
+    compiler: sphinx
+    env: PYTHON=2.7 DOCS=1
+    script: $SCRIPT_RUN_PREFIX make -C docs html SPHINX_OPTIONS=-W
```

Comment:
Why not add `install:` here? Should be a little cleaner and avoid another condition in the main script. Should be about 3 lines: pip install, venv, sphinx.